### PR TITLE
Add LatexDraw.app v3.3.5

### PR DESCRIPTION
**Pull request by ushu**
Here is a Cask definition for LaTexDraw, a small tool for drawing figure in LaTeX documents: thesis, reports, etc. without having to "hand-code" everything.

I run all the required tests before committing 🤓:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the stable versions repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[ stable versions repo]: https://github.com/caskroom/homebrew-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/latexdraw.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'latexdraw' do
+  version '3.3.5'
+  sha256 '905da2d258baf777dff9cb14468207dff04094049f9bc82f050d8a8541967965'
+
+  url "https://downloads.sourceforge.net/latexdraw/LaTeXDraw-#{version}.app.zip"
+  name 'LaTexDraw'
+  homepage 'http://latexdraw.sourceforge.net/'
+
+  app 'LaTexDraw-3.3.5.app'
```
Comment (vitorgalvao) [inline Casks/latexdraw.rb]: [`target:` Should Only Be Used in Select Cases](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

File: Casks/latexdraw.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'latexdraw' do
+  version '3.3.5'
+  sha256 '905da2d258baf777dff9cb14468207dff04094049f9bc82f050d8a8541967965'
+
+  url "https://downloads.sourceforge.net/latexdraw/LaTeXDraw-#{version}.app.zip"
+  name 'LaTexDraw'
```
Comment (vitorgalvao) [inline Casks/latexdraw.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Welcome, and thank you.

Next time, please leave the sections of the template intact. They help visually separate what was done.

File: Casks/latexdraw.rb (hunk 1 above)
Comment (commitay) [inline Casks/latexdraw.rb]: `app "LaTexDraw-#{version}.app"`

File: Casks/latexdraw.rb (hunk 2 above)
Comment (ushu) [inline Casks/latexdraw.rb]: ok I'm currently looking into it !

**Comment by ushu** [comment]
just added the `appcast` call, is that it ?

**Review by vitorgalvao — APPROVED** [review]

**Comment by ushu** [comment]
Hello there,

I don't get the CI response: I followed [the instructions](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) for generating the appcast checkpoint but got a different hash value:

```bash
$ brew cask _appcast_checkpoint --calculate "https://sourceforge.net/projects/latexdraw/rss"
359991ab747f45b0157314d4826dc2cb3f311e222e6f3dca4fff361cd9d35144
```

I can sill amend the code to make the CI happy, but without understanding why it feels kinda bad...

**Comment by giannitm** [comment]
@ushu The appcast `checkpoint` seems to have changed again since your last post. We should investigate why it keeps changing so regularly before we merge this pull request.
```
1 file inspected, no offenses detected
==> Downloading https://downloads.sourceforge.net/latexdraw/LaTeXDraw-3.3.5.app.
######################################################################## 100.0%
==> Verifying checksum for Cask latexdraw
audit for latexdraw: warning
 - appcast checkpoint mismatch
Expected: 359991ab747f45b0157314d4826dc2cb3f311e222e6f3dca4fff361cd9d35144
Actual: 619b027e14b1b2814f18fcb86d1f51839dce5d5c50212884ee565a10558b962a
Error: audit failed for casks: latexdraw
```

**Comment by vitorgalvao** [comment]
Tried an appcast a bit more specific. Let’s see if it sticks.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**